packages/react/src/components/Tag/DismissibleTag.tsx

```diff
@@ -45,6 +46,11 @@ export interface DismissibleTagBaseProps {
    */
   disabled?: boolean;
 
+  /**
+   * Specify the tooltip alignment for the dismiss button
+   */
+  dismissTooltipAlign?: PopoverAlignment;
```

Comment:
Convention across the codebase looks more centered on the name `tooltipAlignment`. With the `dismissTooltipLabel` prop already existing though it sort-of creates a conflicting convention.

I wish both these were `tooltipLabel` and `tooltipAlignment` but to avoid churn on the existing prop we could instead just split it down the middle and do `dismissTooltipAlignment`. What do you think?

```suggestion
  dismissTooltipAlignment?: PopoverAlignment;
```

Reply:
Yeah that makes sense, I've renamed `dismissTooltipAlign` to `dismissTooltipAlignment` and `dismiss-tooltip-align` to `dismiss-tooltip-alignment`